vm/Cargo.toml

```diff
@@ -71,7 +71,7 @@ smallbox = "0.8"
 bstr = "0.2.12"
 crossbeam-utils = "0.7"
 generational-arena = "0.2"
-parking_lot = { git = "https://github.com/Amanieu/parking_lot" }
+parking_lot = { git = "https://github.com/coolreader18/parking_lot", branch="raw-re-is_locked" }
```

Comment:
I was a bit iffy when we added the `parking_lot` from Github. For now at least add a TODO to change this.

Reply:
That was just temporary, the PR for that is merged now. I'll still add a TODO for the official git version, though.